src/main/kotlin/org/polystat/j2eo/treeMapper/TreeMappings.kt

```diff
@@ -4,7 +4,7 @@ package org.polystat.j2eo.treeMapper
 import lexer.Token
 import lexer.TokenCode
 import org.antlr.v4.runtime.tree.TerminalNode
-import org.polystat.j2eo.antlrParser.JavaParser.*
+import JavaParser.*
```

Comment:
@includealex Using start-import is a bad practice. I believe it's related to Konlin as well as to Java.